README.md

```diff
@@ -12,15 +12,15 @@ Having a stream interface to processes is useful when we have multiple sources o
 we want to merge and start processing from a single entry point.
 
 This crate provides a [`core::stream::Stream`] wrapper for [`tokio::process::Child`]. The
-main struct is [`ProcessStream`], which implements the trait, yielding one [`Item`] enum at
+main struct is [`ProcessLineStream`], which implements the trait, yielding one [`Item`] enum at
```

Comment:
I thought it made sense to rename `ProcessStream`, but we could just keep it as is alongside the new `ProcessChunkStream`.

Reply:
Makes sense, I might do that later on